pkg/controller/controller.go

```diff
@@ -207,11 +198,26 @@ func (p *csiProvisioner) Provision(options controller.VolumeOptions) (*v1.Persis
 	capacity := options.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]
 	volSizeBytes := capacity.Value()
 
+	controllerCreateSecrets := map[string]string{}
+	secret := v1.SecretReference{}
+	// get secrets if StorageClass specifies it
+	if sn, ok := options.Parameters[secretNameKey]; ok {
+		ns := "default"
+		if len(options.Parameters[secretNamespaceKey]) != 0 {
+			ns = options.Parameters[secretNamespaceKey]
+		}
+		controllerCreateSecrets = getCredentialsFromSecret(p.client, sn, ns)
+		secret.Name = sn
+		secret.Namespace = ns
+	}
+	fmt.Printf("><SB> Parameters: %+v\n", controllerCreateSecrets)
```

Comment:
Don't forget to remove Printf

Reply:
Just for debugging purposes, will go away..